pysmt/utils.py

```diff
@@ -45,3 +46,22 @@ def twos_complement(val, bits):
     if (val & (1 << (bits - 1))) != 0: # if sign bit is set
         val = val - (2 ** bits)        # compute negative value
     return val
+
+
+_simple_symbol_prog = re.compile(r"^[~!@\$%\^&\*_\-+=<>\.\?\/A-Za-z][~!@\$%\^&\*_\-+=<>\.\?\/A-Za-z0-9]*$")
+def quote(name, style='|'):
```

Comment:
Not a huge fan of the ```utils``` module. These are functions related to serialization/string conversion of symbols and expressions. I think this should go somewhere in the printing modules or directly in "FNode".